### PR TITLE
Created two programs using the concept of inheritance. Calculators (en…

### PR DESCRIPTION
**Pull request by valeriiashyshko**
…gineering, programmable) and Shapes (circle, triangle, rectangle)

### CODE REVIEW

File: src/homework7/calculators/EngineeringCalculator.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package homework7.calculators;
+
+public class EngineeringCalculator extends Calculator {
+
+    public static double squareRoot(double a) {
```
Comment (AndrewSkoba) [inline src/homework7/calculators/EngineeringCalculator.java]: What is the reason to use static?

File: src/homework7/calculators/ProgrammableCalculator.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package homework7.calculators;
+
+public class ProgrammableCalculator extends Calculator {
+
+    public static double tangent(double b) {
+        return Math.tan(b);
+    }
+    public static double sine(double b) {
```
Comment (AndrewSkoba) [inline src/homework7/calculators/ProgrammableCalculator.java:9]: Empty row between methods

File: src/homework7/calculators/EngineeringCalculator.java (hunk 1 above)
Comment (valeriiashyshko) [reply src/homework7/calculators/EngineeringCalculator.java]: Deleted static

File: src/homework7/calculators/ProgrammableCalculator.java (hunk 2 above)
Comment (valeriiashyshko) [reply src/homework7/calculators/ProgrammableCalculator.java:9]: Added an empty row

**Review by AndrewSkoba — APPROVED** [review]

**Pull request merged**